packages/core/src/codewhispererChat/controllers/chat/controller.ts

```diff
@@ -394,11 +394,23 @@ export class ChatController {
 
                 const prompt = this.promptGenerator.generateForContextMenuCommand(command)
 
-                this.messenger.sendEditorContextCommandMessage(
-                    command.type,
-                    context?.focusAreaContext?.codeBlock ?? '',
-                    triggerID
-                )
+                if (command.type === 'aws.amazonq.explainIssue' || command.type === 'aws.amazonq.fixIssue') {
```

Comment:
off topic: command.name or command.id would be a more appropriate name.
```suggestion
                if (command.id === 'aws.amazonq.explainIssue' || command.id === 'aws.amazonq.fixIssue') {
```